lib/node_modules/@stdlib/plot/unicode/static_plots/marker/colors.js

```diff
@@ -0,0 +1,33 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+var colors = {
```

Comment:
Notice the documentation in this file: https://github.com/stdlib-js/stdlib/blob/aab2efb0b25413de93eb8cc0695f752f06e7e346/lib/node_modules/%40stdlib/repl/presentation/lib/ansi_escape_codes.js#L31